tests/Fixtures/TestBundle/State/UserResetPasswordProcessor.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Fixtures\TestBundle\State;
+
+use ApiPlatform\State\ProcessorInterface;
+use ApiPlatform\Tests\Fixtures\TestBundle\Dto\UserResetPasswordDto;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class UserResetPasswordProcessor implements ProcessorInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function resumable(?string $operationName = null, array $context = []): bool
+    {
+        return false;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
```suggestion
```

Reply:
done in commit eada6c0bc00762ec50a0c4a2b9889df3bcbf65ae